### PR TITLE
Display company values

### LINKED ISSUE #83: Add a view for seeing Company Values
As a user, I should be able to: 
- [x] See all of the currently active company values and their descriptions: 
- [x] See all-time grand total for each value listed
- [x] See the top user of a value for shoutouts given
- [x] See the top user of a value for shoutouts received

### PR DESCRIPTION
**Pull request by tgeeTechtonic**
### Type of change:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling - no new features

### Why is this change required? What problem(s) does it solve?
There should be a page where a user can see the company values and some data regarding what they are and how often they are used

### Link issues
Closes #83

### CODE REVIEW

File: client/src/Views/CompanyValues/CompanyValues.vue [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    this.loading = true;
+    await this.$store.dispatch('getCompanyValues');
+    this.loading = false;
+    this.values = companyValuesFormatter(this.$store.state.companyValues);
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../assets/styles/variables.scss' as v;
```
Comment (drogersTechtonic) [inline client/src/Views/CompanyValues/CompanyValues.vue:39]: I see you imported this here as `v` but the variable isn't used anywhere. Is this necessary?

File: client/src/shared/formatters.js [hunk 2]
```diff
@@ -55,3 +55,25 @@ export const capitalizeWordFormatter = (str) => {
...
+    }) => {
+      return {
+        value,
+        description,
+        num_times_used,
+        top_user_shoutouts_given,
+        top_user_shoutouts_received,
+        active,
+      };
+    }
```
Comment (drogersTechtonic) [inline client/src/shared/formatters.js:77]:
```suggestion
    }) => ({
        value,
        description,
        num_times_used,
        top_user_shoutouts_given,
        top_user_shoutouts_received,
        active
    })
```

File: server/controllers/valuesController.js [hunk 3]
```diff
@@ -1,11 +1,55 @@
 const { db } = require('../dbConfig');
 
 const valuesController = {
```
Comment (drogersTechtonic) [inline server/controllers/valuesController.js:3]: Not a big deal, but I'd say consider adding "company" where there's only values now. It's not hard to figure out what this does given the context but isn't clear at a quick glance.

File: server/controllers/valuesController.js [hunk 4]
```diff
@@ -1,11 +1,55 @@
...
+    const num_times_used = await db('shoutouts')
+      .where('company_value', values[i].id)
+      .count();
+    const top_shoutee = await getTopUser(values[i].id, 'shoutee');
+    const top_shouter = await getTopUser(values[i].id, 'shouter');
+
+    results.push({
+      active: values[i].active_status,
+      description: values[i].description,
+      num_times_used: num_times_used[0].count,
```
Comment (drogersTechtonic) [inline server/controllers/valuesController.js:29]: Why snakecase instead of camelcase?

**Review by drogersTechtonic — APPROVED** [review]
Look good ✅ . I left a few comments, none of which were of major concern though, so I'll leave it to you to decide whether to make the changes

File: client/src/Views/CompanyValues/CompanyValues.vue (hunk 1 above)
Comment (tgeeTechtonic) [reply client/src/Views/CompanyValues/CompanyValues.vue:39]: I did it for later, I want to add a banner of some kind to the page as well, but I didn't really know how it should look and it wasn't part of the ticket soo... future planning unused variable?

File: server/controllers/valuesController.js (hunk 3 above)
Comment (tgeeTechtonic) [reply server/controllers/valuesController.js:3]: 👍

File: server/controllers/valuesController.js (hunk 4 above)
Comment (tgeeTechtonic) [reply server/controllers/valuesController.js:29]: From everything I've seen, server side is usually snake case while front end is camel case. I'm not sure that there's a rule, but I like the distinction personally

**Pull request merged**